Project.toml

```diff
@@ -15,6 +15,7 @@ LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
+LinearOperators = "5c8ed15e-5a4c-59e4-a42b-c7e8811fb125"
```

Comment:
To appease the Julia package bot, we need to add a corresponding entry to the `[compat]` section.